bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump.properties

```diff
@@ -0,0 +1,132 @@
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile1.state.option.1 = Heat pump stopped
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile1.state.option.2 = Heat pump coming
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile1.state.option.3 = Error code memory location 0
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile1.state.option.4 = Defrost
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile1.state.option.5 = Wait for LIN connection
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile1.state.option.6 = Compressor heats up
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile1.state.option.7 = Pump flow
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile2.state.option.0 = since
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile2.state.option.1 = in
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.0 = Heating mode
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.1 = No requirement
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.2 = Mains switch-on delay
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.3 = Switching cycle lock
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.4 = Blocking time
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.5 = Service water
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.6 = Info bakeout program
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.7 = Defrost
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.8 = Pump flow
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.9 = Thermal disinfection
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.10 = Cooling mode
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.12 = Swimming pool / Photovoltaics
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.13 = Heating ext. energy source
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.14 = Service water ext. energy source
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.16 = Flow monitoring
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.17 = Second heat generator 1 operation
+c
+
+channel-type.luxtronikheatpump.Ba_XX_akt.state.option.0 = Auto
+channel-type.luxtronikheatpump.Ba_XX_akt.state.option.1 = Auxiliary heater
+channel-type.luxtronikheatpump.Ba_XX_akt.state.option.2 = Party
+channel-type.luxtronikheatpump.Ba_XX_akt.state.option.3 = Holiday
+channel-type.luxtronikheatpump.Ba_XX_akt.state.option.4 = Off
+
+channel-type.luxtronikheatpump.SwitchoffX_file_NrX.state.option.1 = Heat pump malfunction
+channel-type.luxtronikheatpump.SwitchoffX_file_NrX.state.option.2 = Facility malfunction
+channel-type.luxtronikheatpump.SwitchoffX_file_NrX.state.option.3 = Second heat generator operating mode
+channel-type.luxtronikheatpump.SwitchoffX_file_NrX.state.option.4 = EVU lock
```

Comment:
Is EVU English?

Reply:
I guess `EVU` stands for `Energieversorgungsunternehmen`, not sure if there is a short equivalent in english. The heatpump uses `EVU` for english as well :thinking:

Reply:
Tried to find something with the same meaning on the internet. Seems "Utility lock" might work. So will use this for now